src/components/ActorColumn.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Character from './Character.jsx'
+import { Droppable } from 'react-beautiful-dnd';
+
+// Individual actor column that has a droppable area and displays all the characters assigned to that actor
+const ActorColumn = ({column, characterList}) => {
+  return (
+    <div className='columnContainer' style={{height :'fit-content'}}>
```

Comment:
might consider making the class name more specific to the drag and drop such as "dropContainer", "actorDropColumn", etc.